lib/reports/resolvers.ts

```diff
@@ -237,15 +249,35 @@ export const generateApplicationsReport: Resolver<
     ({ name, reportColumnId }) => ({ id: reportColumnId, title: name })
   );
 
-  const csvWriter = createObjectCsvWriter({
-    path: 'temp/file.csv',
+  // Generate CSV string from csv object.
+  const csvStringifier = createObjectCsvStringifier({
     header: csvHeaders,
   });
+  const csvStringRecords = csvStringifier.stringifyRecords(csvApplications);
+  const csvStringHeader = csvStringifier.getHeaderString();
+  const csvString = csvStringHeader + csvStringRecords;
 
-  await csvWriter.writeRecords(csvApplications);
+  // CSV naming format applications/permit-holders/accounting-report-{employeeID}-{timestamp}.csv
+  const employeeID = session.id;
+  const timestamp = formatDateTimeYYYYMMDDHHMMSS(new Date());
+  const fileName = `accounting-report-${employeeID}-${timestamp}.csv`;
+  const s3InvoiceKey = `rcd/applications/permit-holders/${fileName}`;
```

Comment:
```suggestion
  const fileName = `accounting-report-${employeeID}-${timestamp}.csv`;
  const s3ObjectKey = `rcd/reports/${fileName}`;
```
Sorry, by `applications/permit-holders/accounting` I meant whichever was applicable to the type of report

Reply:
🤦 my bad